storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -87,12 +89,17 @@ include::modules/lvms-creating-lvms-cluster-using-web-console.adoc[leveloffset=+
 
 * xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
 
+include::modules/lvms-creating-lvmcluster-using-rhacm.adoc[leveloffset=+2]
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
```

Comment:
Probably want the Additional resources header here:
```suggestion
role="_additional-resources"]
.Additional resources

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
```

Reply:
Updated as per the suggestion.